Fix error #1145 nrf_jammer.cpp

LittleHackid opened the pull request:
#### Proposed Changes ####
I put these lines of code in the jammer and channel selection section. 
tft.setCursor(10,100);
uint16_t bgColor = ILI9341_BLACK;
tft.fillRect(10, 100, 250, 16, bgColor);

LittleHackid committed:
Fix error #1145 nrf_jammer.cpp

I put these lines of code in the jammer and channel selection section.
tft.setCursor(10,100);
uint16_t bgColor = ILI9341_BLACK;
tft.fillRect(10, 100, 250, 16, bgColor);

bmorcelli commented on src/modules/NRF24/nrf_jammer.cpp:
```diff
@@ -64,6 +64,9 @@ void nrf_jammer() {
                 tft.println("STATUS : ACTIVE");
+                tft.setCursor(10,100);
+                uint16_t bgColor = ILI9341_BLACK;
+                tft.fillRect(10, 100, 250, 16, bgColor);
```
you don't need to set cursor again, and the color it would be better use the background color used by the theme:

use only this in both pplaces:
```
tft.fillRect(10, 100, 250, 16, bruceConfig.bgColor);
```

bmorcelli commented on src/modules/NRF24/nrf_jammer.cpp:
```diff
@@ -183,6 +186,9 @@ void nrf_channel_jammer() {
                 tft.print("MODE : " + NRF_MODE + "  ");
+                tft.setCursor(10,100);
+                uint16_t bgColor = ILI9341_BLACK;
+                tft.fillRect(10, 100, 250, 16, bgColor);
```
you don't need to set cursor again, and the color it would be better use the background color used by the theme:

use only this in both pplaces:
```
tft.fillRect(10, 100, 250, 16, bruceConfig.bgColor);
```

LittleHackid committed:
Update nrf_jammer.cpp

The creator made a change instead of my original proposal.
tft.fillRect(10, 100, 250, 16, bruceConfig.bgColor);
rathen than...
tft.setCursor(10,100);
uint16_t bgColor = ILI9341_BLACK;
tft.fillRect(10, 100, 250, 16, bgColor);
